sql/bgw_scheduler.sql

```diff
@@ -21,7 +21,11 @@ INSERT INTO _timescaledb_config.bgw_job (id, application_name, job_type, schedul
 (1, 'Telemetry Reporter', 'telemetry_and_version_check_if_enabled', INTERVAL '24h', INTERVAL '100s', -1, INTERVAL '1h')
 ON CONFLICT (id) DO NOTHING;
 
-CREATE OR REPLACE FUNCTION add_drop_chunks_policy(hypertable REGCLASS, older_than INTERVAL, cascade BOOL = FALSE, if_not_exists BOOL = false, cascade_to_materializations BOOL = false)
+CREATE OR REPLACE FUNCTION set_integer_now_func(hypertable REGCLASS, integer_now_func REGPROC, replace_if_exists BOOL = false) RETURNS VOID
+AS '@MODULE_PATHNAME@', 'ts_set_integer_now_func'
+LANGUAGE C VOLATILE STRICT;
+
+CREATE OR REPLACE FUNCTION add_drop_chunks_policy(hypertable REGCLASS, older_than "any", cascade BOOL = FALSE, if_not_exists BOOL = false, cascade_to_materializations BOOL = false)
```

Comment:
We have to actually drop the old function if it exists, either here or probably in the update script as otherwise it will be overloaded in a weird way and may break things horribly.

Reply:
Will put it in the update script.
 I think Create or replace fails if the function exists with a different signature so worst case it will fail but not result in an unexpected state